.kokoro/downstream-client-library-check.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -eo pipefail
+# Display commands being run.
+set -x
+
+CORE_LIBRARY_ARTIFACT=$1
+CLIENT_LIBRARY=$2
```

Comment:
Do you think this script `.kokoro/downstream-client-library-check.sh` is going to be managed by Owlbot in future? (Not required but that will be nice)

Reply:
Since this script is only required by less than 10 libraries and there are 140+ client libraries managed by owlbot, it will adding an extra not-required file to other client libraries with the current owlbot design.
May be we can consider adding an exclude option for templated files in owlbot.